.changeset/new-radios-pay.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': patch
+---
+
+Improve static asset generation performance by removing the `await` in the `for` loop. This change will allow the p-queue concurrency to limit the number of parallel tasks. Previously, the `await` in the `for` loop would cause all asset generation tasks to run serially.
```

Comment:
For future reference, these are used in the changelog, so are user-facing rather than technical implementation details. They're phrased as if they start with "This PR...", so in this case it would be "Improves static asset..." etc. I can do the re-wording for this.